Dockerfile

```diff
@@ -2,20 +2,16 @@ FROM nginx:1.23.3-alpine
 
 LABEL maintainer="fehguy"
 
-ENV BASE_URL ""
+ENV BASE_URL "/" \
```

Comment:
Can we change the `NGINX_PORT` TO `PORT`? (Just for consistency with SwaggerUI docker image)

Reply:
> According to the [docker docs](https://docs.docker.com/engine/reference/builder/#env) this should be defined as following:
> 
> ```
> ENV BASE_URL="/" \
>     NGINX_PORT="8080"
> ```
> 
> Can we change this env defs so that it's aligned with official docker docs?

But this is exactly how it is defined in line 5 and 6. Or am I missing something?

Reply:
```
ENV BASE_URL "/" \
    NGINX_PORT "8080"
```
vs 
```
ENV BASE_URL="/" \
    NGINX_PORT="8080"
```

Reply:
Thanks, I didn't noticed it.
Now `ENV`s and `PORT` are applied.